dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -220,11 +214,12 @@ static Configuration configureFreemarker() {
       Configuration configuration,
       Path output,
       ArtifactCache cache,
-      ImmutableSetMultimap<SymbolProblem, ClassFile> symbolProblems,
+      ImmutableMap<Path, ImmutableSetMultimap<SymbolProblem, String>> symbolProblemTable,
       ListMultimap<Path, DependencyPath> jarToDependencyPaths) {
 
     ImmutableSetMultimap<String, Path> bomMemberToJars = bomMemberToJars(jarToDependencyPaths);
 
+
```

Comment:
Delete extra blank line.

Have you run google-java-format?

Reply:
I think that option looks only at changes that haven't been committed yet, compared to the last commit. There really should be a way of looking only at lines that have been changed on this branch, compared to master.

Aha! From [google-java-format.diff.py](https://github.com/google/google-java-format/blob/master/scripts/google-java-format-diff.py):

```shell
$ git diff -U0 master...HEAD | google-java-format-diff.py -p1 -i
```

That should use the diff between (the common ancestor of master and the current branch) and HEAD to determine which lines to format. Note the three dots.

Reply:
Thanks. Installed the script but it seems touching other files than the diff. Will show you later.

Reply:
It worked. This is what I needed:

    ~/cloud-opensource-java$ git fetch; git diff -U0 origin/master...HEAD | google-java-format-diff.py -p1 --skip-sorting-imports -i